libs/experimental/langchain_experimental/llms/ollama_functions.py

```diff
@@ -151,33 +148,13 @@ def bind_tools(
     ) -> Runnable[LanguageModelInput, BaseMessage]:
         return self.bind(functions=tools, **kwargs)
 
-    @overload
```

Comment:
why remove the overloads?

in fact, could we move these to base chat model?

Reply:
Adding those overloads to the base class causes tons of lint errors in already implemented classes. Because of that, I'm going to not add it since I'm worried about breaking typing for existing users.